exercises/simple-linked-list/src/LinkedList.hs

```diff
@@ -0,0 +1,41 @@
+module LinkedList
+    ( LinkedList
+    , datum
+    , fromList
+    , isNil
+    , new
+    , next
+    , nil
+    , reverseLinkedList
+    , toList
+    ) where
+
+-- The task is to create the data type `LinkedList`
```

Comment:
hmm, if the task is to create the list, do you intend to include the `data LinkedList a` in the stub?